tools/block_cache_pysim.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
```

Comment:
Curious why not use python script, but you don't have to change it. :)

Reply:
I'm not sure how effective is the python multi-threading so I used the bash script to start multiple processes to speed up simulation.